generator/source/faq/installation.rst

```diff
@@ -256,6 +256,8 @@ The usual problem when you get this message is configuring the peer endpoints
 
 * Check peer configuration in your local and remote ``/etc/sawtooth/validator.toml`` files. Check the ``peering`` and ``endpoint`` lines. Check the ``seeds`` line (for dynamic peering) or ``peers`` line (for static peering)
 
+* If you are connecting to external nodes on the network, bind to 0.0.0.0 (which means any IP address) or your external IP address instead of 127.0.0.1 (localhost only).
```

Comment:
It might be worth a comment here that this has security implications.

Reply:
Worth mentioning example configuration. Many people do that mistake of leaving "--endpoint tcp://non-public-address:8800" when running in docker containers but on multiple machines.

Reply:
Good point. I added:

> Be careful doing this as this opens up the listening port to the outside network.

Reply:
I would prefer this to say "(which means every IP address)" and "Be careful doing this as it can potentially open up this port to external networks".

Reply:
Reworded to:
> Be careful doing this as may open up this port to external networks.